packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -844,7 +844,8 @@ AWS CloudWatch for all API calls:
 
 > By default, an IAM role will be created and associated with API Gateway to
 allow it to write logs and metrics to AWS CloudWatch unless `cloudWatchRole` is
-set to `false`.
+set to `false`. To change this default behavior you can enable the feature flag
+`@aws-cdk/aws-apigateway:disableCloudWatchRole`
```

Comment:
Double checking - is this how we want to phrase feature flagged defaults? All new apps will have the flag enabled and it might confuse people. Maybe it'd be better to directly call out that the FF in the first sentence?!

Reply:
Good point. Just pushed an update, let me know what you think or feel free to suggest different wording.